frontend/packages/dev-console/src/components/import/route/RouteSection.tsx

```diff
@@ -1,25 +1,13 @@
 import * as React from 'react';
-import { useTranslation } from 'react-i18next';
-import FormSection from '../section/FormSection';
-import { RouteData } from '../import-types';
 import CreateRoute from './CreateRoute';
 import SecureRoute from './SecureRoute';
 
-interface RouteSectionProps {
-  route: RouteData;
-}
-
-const RouteSection: React.FC<RouteSectionProps> = ({ route }) => {
-  const { t } = useTranslation();
+const RouteSection: React.FC = () => {
```

Comment:
@vikram-raj I dont think we need this wrapper component anymore, You can directly use CreateRoute, SecureRoute in the AdvancedRouteOption component

Reply:
Removed this wrapper component.